genanki/deck.py

```diff
@@ -13,22 +13,52 @@ def add_note(self, note):
     self.notes.append(note)
 
   def add_model(self, model):
-    self.models[model.model_id] = model
-
-  def write_to_db(self, cursor, now_ts):
-    if not isinstance(self.deck_id, int):
-      raise TypeError('Deck .deck_id must be an integer, not {}.'.format(self.deck_id))
-    if not isinstance(self.name, str):
-      raise TypeError('Deck .name must be a string, not {}.'.format(self.name))
+    self.models[model.id] = model
 
+  def update_models(self):
```

Comment:
Make this private (preface with underscore). Although I'm not sure it really warrants splitting out a method anyway.